(DOCSP-21946): Incorporate feedback from tech review on PR #1810 (#1840)
mongodb/docs-realm: source/sdk/swift/advanced-guides/client-reset.txt

      :language: swift

.. _ios-manual-client-reset-handler:
.. _ios-manual-mode:

Manual Client Reset
-------------------

When you specify ``.manual`` for ``.clientResetMode``, or if you do not 

> specify a client reset mode, you can implement your own client reset handler.

Reviewer: ```suggestion
specify a client reset mode, you should implement your own client reset handler.
```

Can, or should? I believe most users should implement a client reset handler, so maybe this is a good opportunity to be prescriptive?

If your app uses 10.24.2 or earlier, ``.clientResetMode`` is not an 
available property on the SyncConfiguration. Older SDK versions use the manual 
client reset handler.

> In ``.manual`` mode, the local realm file is copied into a recovery directory, 

Reviewer: ```suggestion
In ``.manual`` mode, the SDK copies the local realm file into a recovery directory, 
```
Passive -> active voice

> and deleted from its original location. The next time your app opens the realm 

Reviewer: ```suggestion
and deletes the original copy of the realm. The next time your app opens the realm 
```

Slightly clearer + consistent voice with my previous suggestion

Author: I like it - taking a modified version:
> and deletes the original.

"Original copy" is a little weird and potentially confusing, given that we're talking about copying stuff around here.

> for that partition, you can re-download the realm and use it as normal. You 

Reviewer: ```suggestion
for that partition, the SDK downloads a new copy of the realm. This new copy can now synchronize with the backend. You 
```

I feel like it's confusing to make it seem like the user has to re-download the realm when the SDK handles that automatically on next open. Also separating out the "now syncable" thought into a separate sentence, since it's easy to miss.
